src/App.js

```diff
@@ -28,13 +28,13 @@ class App extends React.Component {
 
 const Conditional = () => {
   const { user, isAuthenticated } = useAuth();
-  const [userInDB, setUserInDB] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
oooh hooks, fancy :smirk:

Reply:
yee now i actually sorta understand them, and not just copying from tutorials lol